README.md

```diff
@@ -37,174 +38,166 @@ Rust, Cargo, Crates.io, or the RFC process itself. What constitutes a
+the Rust community's [sub-team] specific guidelines for:
 
-* [language changes](lang_changes.md),
-* [library changes](libs_changes.md),
-* [compiler changes](compiler_changes.md).
+  - [Language Changes](lang_changes.md)
+  - [Library Changes](libs_changes.md)
+  - [Compiler Changes](compiler_changes.md)
 
 
 ## Before creating an RFC
 [Before creating an RFC]: #before-creating-an-rfc
 
 A hastily-proposed RFC can hurt its chances of acceptance. Low quality
-proposals, proposals for previously-rejected features, or those that
-don't fit into the near-term roadmap, may be quickly rejected, which
-can be demotivating for the unprepared contributor. Laying some
-groundwork ahead of the RFC can make the process smoother.
+proposals, proposals for previously-rejected features, or those that don't fit
+into the near-term roadmap, may be quickly rejected, which can be demotivating
+for the unprepared contributor. Laying some groundwork ahead of the RFC can
+make the process smoother.
 
-Although there is no single way to prepare for submitting an RFC, it
-is generally a good idea to pursue feedback from other project
-developers beforehand, to ascertain that the RFC may be desirable:
-having a consistent impact on the project requires concerted effort
-toward consensus-building.
+Although there is no single way to prepare for submitting an RFC, it is
+generally a good idea to pursue feedback from other project developers
+beforehand, to ascertain that the RFC may be desirable: having a consistent
+impact on the project requires concerted effort toward consensus-building.
 
-The most common preparations for writing and submitting an RFC include
-talking the idea over on #rust-internals, filing and discusssing ideas
-on the [RFC issue tracker][issues], and occasionally posting
-'pre-RFCs' on [the developer discussion forum][discuss] for early
-review.
+The most common preparations for writing and submitting an RFC include talking
+the idea over on #rust-internals, filing and discussing ideas on the
+[RFC issue tracker], and occasionally posting "pre-RFCs" on the
+[developer discussion forum] for early review.
 
-As a rule of thumb, receiving encouraging feedback from long-standing
-project developers, and particularly members of the relevant [sub-team]
-is a good indication that the RFC is worth pursuing.
-
-[issues]: https://github.com/rust-lang/rfcs/issues
-[discuss]: http://internals.rust-lang.org/
+As a rule of thumb, receiving encouraging feedback from long-standing project
+developers, and particularly members of the relevant [sub-team] is a good
+indication that the RFC is worth pursuing.
 
 
 ## What the process is
 [What the process is]: #what-the-process-is
 
-In short, to get a major feature added to Rust, one must first get the
-RFC merged into the RFC repo as a markdown file. At that point the RFC
-is 'active' and may be implemented with the goal of eventual inclusion
```

Comment:
I was only attempting to increase consistency of punctuation characters. (consistent-punctuation)